packages/sit-onyx/src/components/OnyxDataGrid/features/index.ts

```diff
@@ -455,6 +460,10 @@ export const useDataGridFeatures = <
   const watchSources: WatchSource[] = features.flatMap((f) => f.watch ?? []);
 
   return {
+    /**
+     * Takes the table attributes and maps all
+     */
+    createRendererTableAttributes,
```

Comment:
Suggestion: We change this API to create attributes for the scroll container and pass them through using a new property `scrollContainerArgs` on the `OnyxDataGridRenderer`
```suggestion
    createScrollContainerAttributes,
```